Sources/PackageModel/Module.swift

```diff
@@ -126,7 +126,7 @@ extension ClangModule: XcodeModuleProtocol {
 
 extension ClangModule: ModuleTypeProtocol {
     public var mainFiles: Set<String> {
-        return ["main.c", "main.m"]
+        return ["main.c", "main.m", "main.cpp"]
```

Comment:
This should be consistent with the full set of extensions we support, but see comment in previous PR.